features/place/duplicate.feature

```diff
@@ -11,9 +11,15 @@ Feature: Test creating places
     Given I create a random name of 6 characters and keep it as "name"
     Given I create a minimal place and save the "id" as "originalPlaceId" then I should get a "201" response code
     Then I wait for the place with url "/places/%{originalPlaceId}" to be indexed
-    Given I create a minimal place and save the "originalPlace" as "newPlaceUri" then I should get a "409" response code
-    Then the JSON response at "originalPlace" should be:
+    Given I create a minimal place and save the "query" as "query" then I should get a "409" response code
```

Comment:
Is it still needed to save this `query` value? I get the feeling that we could do without this extra custom step by just calling the existing steps like, for example:

```
When I send a POST request to "/places/"
Then the response status should be "409"
And I keep the value of the JSON response at "id" as "originalPlaceId"
```